Misc/NEWS.d/next/Core and Builtins/2023-09-06-13-28-42.gh-issue-108732.I6DkEQ.rst

```diff
@@ -0,0 +1,2 @@
+Make iteration variables of module- and class-scoped comprehensions visible
```

Comment:
Maybe mention `frame.f_locals` here explicitly? pdb might be the main place this comes up within CPython, but I'm sure other tools would be able to run into it too.